system.go

```diff
@@ -11,12 +11,24 @@ import (
 )
 
 func countOpenFiles() int {
-	out, err := exec.Command("/bin/sh", "-c", fmt.Sprintf("lsof -p %v", os.Getpid())).Output()
-	if err != nil {
-		log.Fatal(err)
+
```

Comment:
```suggestion
```
You have to buy me a coffee soon every time you have this extra line in a PR :wink:

Reply:
Doesn't go fmt take care of this.i hope you spent less time reviewing this commit :)